core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/OrientedRectangle2afp.java

```diff
@@ -57,6 +57,22 @@
 		V extends Vector2D<? super V, ? super P>,
 		B extends Rectangle2afp<?, ?, IE, P, V, B>>
 		extends Parallelogram2afp<ST, IT, IE, P, V, B> {
+    /**+
```

Comment:
OK. Is the indentation ERROR signalled by the CI checking in logs due to
that + sign?

On Mon, May 30, 2016 at 8:48 PM, Stéphane Galland <notifications@github.com>
wrote:

> In
> core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/OrientedRectangle2afp.java
> <https://github.com/gallandarakhneorg/afc/pull/64#discussion_r65096074>:
>
> > @@ -57,6 +57,22 @@
> >  		V extends Vector2D<? super V, ? super P>,
> >  		B extends Rectangle2afp<?, ?, IE, P, V, B>>
> >  		extends Parallelogram2afp<ST, IT, IE, P, V, B> {
> > +    /**+
>
> Please remove the + character in all the comments.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/64/files/7117c6cdb725bf595ff14df245264a82776b179a#r65096074>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BPOW2L-8j-jnL-UkK2trO3yPyOoqks5qGyL8gaJpZM4Iptts>
> .
>